sdk/keyvault/keyvault-admin/samples/v4-beta/javascript/README.md

```diff
@@ -0,0 +1,75 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-key-vault
+urlFragment: keyvault-admin-javascript-beta
+---
+
+# Azure Key Vault Administration client library samples for JavaScript (Beta)
+
+These sample programs show how to use the JavaScript client libraries for Azure Key Vault Administration in some common scenarios.
+
+| **File Name**                                             | **Description**                                                                                       |
+| --------------------------------------------------------- | ----------------------------------------------------------------------------------------------------- |
+| [accessControlHelloWorld.js][accesscontrolhelloworld]     | Uses an AccessControlClient to list, create, and assign roles to users.                               |
+| [backupRestoreHelloWorld.js][backuprestorehelloworld]     | Uses a BackupClient to backup and fully restore an Azure Key Vault using Azure Storage Blob.          |
+| [backupSelectiveKeyRestore.js][backupselectivekeyrestore] | Uses a BackupClient to backup and restore a specific key in Azure Key Vault using Azure Storage Blob. |
+| [updateSettings.js][updatesettings]                       | Demonstrates how to retrieve and update account settings for Managed HSM.                             |
+
+## Prerequisites
+
+The sample programs are compatible with [LTS versions of Node.js](https://github.com/nodejs/release#release-schedule).
+
+You need [an Azure subscription][freesub] and the following Azure resources to run these sample programs:
+
+- [Azure Key Vault][createinstance_azurekeyvault]
+
+To quickly create the needed Key Vault resources in Azure and to receive a connection string for them, you can deploy our sample template by clicking:
+
+[![](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2Fmaster%2Fsdk%2Fkeyvault%2Ftest-resources.json)
+
+If creating the Key Vault manually using the Azure Portal, be aware that the samples require that the soft-delete feature be enabled. Our template above will enable this feature automatically, but it is possible to enable it manually using the Azure CLI. See [the documentation for enabling soft-delete in Key Vault](https://docs.microsoft.com/azure/key-vault/key-vault-soft-delete-cli) for more information.
```

Comment:
soft-delete is no longer optional as of about a year ago and has been the default for a couple years. Please update this text and check the other READMEs as well.

Reply:
This file is auto-generated from somewhere that I didn't touch in this PR, but I can do a quick sweep.